deploy/crs/kieapp_production.yaml

```diff
@@ -5,12 +5,4 @@ metadata:
 spec:
   environment: production
   kieDeployments: 2
-  objects:
-    console:
-      resources:
-        limits:
-          memory: 2Gi
-    server:
-      env:
-        - name: key1
-          value: value1
+  consoleType: "Monitoring"
```

Comment:
Besides what I explained in my previous comment we can add the `consoleType: None` or maybe an additional attribute to the `CustomObject`. e.g. `skipDeployment: true` to tell the Operator to avoid the deployment of such object. This will provide the flexibility to deploy or not deploy each object (Console/Server/Other). This is useful for cases like _Immutable Kieserver_ or _Immutable Monitoring_. What do you think?